iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -99,43 +127,47 @@ class ConversationMessagesCell: UICollectionViewCell {
     func set(isPreparedToSend: Bool) {
         self.dataSource.shouldPrepareToSend = isPreparedToSend
     }
-
```

Comment:
This isn't needed any more because the cell has conversation controller subscription.